fastify.js

```diff
@@ -22,6 +22,73 @@ const ContentTypeParser = require('./lib/ContentTypeParser')
 const Hooks = require('./lib/hooks')
 const loggerUtils = require('./lib/logger')
 
+const ffs = require('fast-fast-series')
+const urlUtil = require('url')
+
+const validation = require('./lib/validation')
+const validateSchema = validation.validate
+
+function onRequestHandling (obj, callback) {
+  this.onRequest(obj.req, obj.res, function (err) {
+    callback(err, obj)
+  })
+}
+
+function parsingGet (obj, callback) {
+  var req = obj.req
+  obj.request = new this.Request(
+    req.params,
+    req,
+    null,
+    urlUtil.parse(req.url, true).query,
+    req.headers,
+    req.log
+  )
+  callback(null, obj)
+}
+
+function validate (obj, callback) {
+  var valid = validateSchema(this.compiledSchema, obj.request)
+  if (valid !== true) {
+    throw new Error('GGG')
```

Comment:
`GGG` ?

Reply:
This is a WIP. I have to know if the team is agree with this PR and the change.